Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -688,3 +688,31 @@ label mas_wrs_reddit:
     if not wrs_success:
         $ mas_unlockFailedWRS('mas_wrs_reddit')
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_twitch",
+            category=['twitch'],
```

Comment:
```suggestion
            category=['-twitch'],
```
I did some testing on this, and using `-twitch` allows it to catch it when you are on any individual streams ('think user - Twitch' for the window name) without it also catching any window that just had `twitch` on it.